spec/static/spec/v0.9/openapitransactional.yaml

```diff
@@ -21,21 +21,86 @@ tags:
 - description: Extension to OGC API - Features to support STAC metadata model and
     search API
   name: STAC
-- description: |
-    All write requests require authentication. The currently available options for a user to
-    authenticate himself are described below.
+- name: Data Management
+  description: |
+    Metadata management requests. Theses requests are used to create, update or delete the STAC
+    metadata.
+
+    *NOTE: these requests requires authentication as described in [here](#tag/Authentication).*
```

Comment:
picky, I know: -s in `requires`

Reply:
Not picky, correctness :-P I'm glad you find my bad english though thanks.